admin/organization/security-settings/domains.md

```diff
@@ -0,0 +1,65 @@
+---
+description: Domain management in Docker Admin
+keywords: domains, SCIM, SSO, Docker Admin, domain audit
+title: Domain management
+---
+
+{% include admin-early-access.md %}
```

Comment:
Not sure if you saw the new styling of early access menus in the Scout docs, but take a look at those if you want the Admin ones to use the purple styling also it has a fun rocketship icon 😄